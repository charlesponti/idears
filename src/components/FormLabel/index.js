import React from 'react';
import styles from './FormLabel.module.scss';

function FormLabel({ children }) {
  return <label className={styles.container}>{children}</label>;
}

export default FormLabel;
