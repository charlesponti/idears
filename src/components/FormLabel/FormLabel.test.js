
import React from 'react'
import { render } from '@testing-library/react'
import FormLabel from './FormLabel'

describe('<FormLabel/>', () => {
  it('should render component', () => {
    const { container } = render(<FormLabel/>)
    expect(container).toMatchSnapshot()
  })
})


import React from 'react'
import { render } from '@testing-library/react'
import FormLabel from './FormLabel'

describe('<FormLabel/>', () => {
  it('should render component', () => {
    const { container } = render(<FormLabel/>)
    expect(container).toMatchSnapshot()
  })
})

