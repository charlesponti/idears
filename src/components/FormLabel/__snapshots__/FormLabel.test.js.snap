// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormLabel/> should render component 1`] = `
<div>
  <label
    class="container"
  />
</div>
`;
